doc/source/whatsnew/v0.25.1.rst

```diff
@@ -157,7 +157,7 @@ ExtensionArray
 Other
 ^^^^^
 
--
+- Add pyqt in RENAME dictionary in `generate_pip_deps_from_conda.py` to fix installation failure of pandas (:issue:`26838`)
```

Comment:
Referencing `generate_pip_deps_from_conda.py` probably isn't relevant to users. Let's just write this as "Fixed pyqt development dependency."

This doesn't need to be back ported, so you can remove the release note or move it to 1.0.0.rst